security/advancedtls/advancedtls.go

```diff
@@ -164,7 +164,7 @@ type ServerOptions struct {
 	// invoke this function every time asked to present certificates to the
 	// client when a new connection is established. This is known as peer
 	// certificate reloading.
-	GetCertificate func(*tls.ClientHelloInfo) (*tls.Certificate, error)
+	GetCertificate func(*tls.ClientHelloInfo) (*[]tls.Certificate, error)
```

Comment:
Let's make this a slice of a pointer of certificates, instead of a pointer to a slice.

Reply:
Sure!